### PR TITLE
bugfix/update-cli

### PR DESCRIPTION
**Pull request by busticated**
https://app.clubhouse.io/particle/story/47341/particle-update-cli-fails-to-properly-install-native-modules-serialport-node-usb

### CODE REVIEW

File: gode/packages.go [hunk 1]
```diff
@@ -134,12 +134,13 @@ func execNpm(args ...string) (string, string, error) {
 func environ() []string {
 	env, path := environPath()
 	env = append(env, "PATH="+prependPathToList(filepath.Dir(nodePath), path))
-	env = append(env, "NPM_CONFIG_ALWAYS_AUTH=false")
-	env = append(env, "NPM_CONFIG_CACHE="+filepath.Join(rootPath, ".npm-cache"))
-	env = append(env, "NPM_CONFIG_REGISTRY="+registry)
-	env = append(env, "NPM_CONFIG_GLOBAL=false")
-	env = append(env, "NPM_CONFIG_ONLOAD_SCRIPT=false")
-	env = append(env, "NPM_CONFIG_AUDIT=false")
+	// TODO (mirande): user lowercase versions - see: https://docs.npmjs.com/misc/config#environment-variables
```
Comment (monkbroc) [inline gode/packages.go]: This seems like a good permanent change. Remove the TODO?

**Review by monkbroc — APPROVED** [review]
Nice work Sherlock!

File: gode/packages.go (hunk 1 above)
Comment (busticated) [reply gode/packages.go]: ah, lol. yes will do :+1:

Comment (busticated) [reply gode/packages.go]: ✅ done

**Pull request merged**